JSDemos/Demos/Scheduler/GroupOrientation/description.md

```diff
@@ -1 +1,3 @@
-This example shows the **workWeek** view with different group orientations. Select the *"Vertical Grouping"* tab to&nbsp;see the resource headers organized in&nbsp;a&nbsp;column and the *"Horizontal Grouping"*&nbsp;&mdash; arranged in&nbsp;a&nbsp;row.
\ No newline at end of file
+If [resources](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/) are assigned to appointments, you can group them by these resources. To do this, specify the [groups](/api-reference/10%20UI%20Components/dxScheduler/1%20Configuration/groups.md '/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#groups') array. Each element of this array is [fieldExpr](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/resources/#fieldExpr) of a resource kind. Note that the order of resource fields depends on the order of elements in the **resources** array.
```

Comment:
Let's look at the description from the standpoint of a reader who doesn't know about resource. We should first introduce him to the concept, give a link to the Resources demo, and only then go on about grouping.